### PR TITLE
qiime-map -> sample-file

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -42,7 +42,6 @@ before_script:
   - cat /tmp/supervisord.log
 script:
   - source activate qp-target-gene
-  - start_target_gene "https://localhost:8383" register ignored
```
Comment (ElDeveloper) [inline .travis.yml:45]: Wouldn't this prevent the plugin from starting?

File: qp_target_gene/split_libraries/util.py [hunk 2]
```diff
@@ -41,7 +43,39 @@ def get_artifact_information(qclient, artifact_id):
...
+    }
+    sort_columns = ['BarcodeSequence', 'LinkerPrimerSequence']
+
+    if 'reverselinkerprimer' in df.columns:
+        rename_cols['reverselinkerprimer'] = 'ReverseLinkerPrimer'
+        sort_columns.append('ReverseLinkerPrimer')
+
+    df.rename(columns=rename_cols, inplace=True)
+    # by design the prep info file doesn't have a Description column so we can
+    # prefil without checking
```
Comment (ElDeveloper) [inline qp_target_gene/split_libraries/util.py:62]:
```suggestion
    # fill without checking
```

File: qp_target_gene/split_libraries/util.py [hunk 3]
```diff
@@ -41,7 +43,39 @@ def get_artifact_information(qclient, artifact_id):
...
+    index = df.index
+    df['Description'] = pd.Series(['XXQIITAXX'] * len(index), index=index)
```
Comment (ElDeveloper) [inline qp_target_gene/split_libraries/util.py:64]: This should work to set this value on all rows:

```suggestion
    df['Description'] = 'XXQIITAXX'
```

File: qp_target_gene/split_libraries/util.py [hunk 4]
```diff
@@ -41,7 +43,39 @@ def get_artifact_information(qclient, artifact_id):
     # Get the artifact metadata
     prep_info = qclient.get('/qiita_db/prep_template/%s/'
                             % artifact_info['prep_information'][0])
-    qiime_map = prep_info['qiime-map']
+
+    df = pd.read_csv(
+        prep_info['prep-file'], sep='\t', index_col=0, dtype='str')
```
Comment (ElDeveloper) [inline qp_target_gene/split_libraries/util.py:48]: Same comments from the other PR regarding data type inference.

**Review by ElDeveloper — CHANGES_REQUESTED** [review]
Looks good overall, just a few minor changes.

File: .travis.yml (hunk 1 above)
Comment (antgonza) [reply .travis.yml:45]: qiita automatically does that, in fact the error that was making the builds fail is that qiita was complaining that the commands already existed ...

File: qp_target_gene/split_libraries/util.py (hunk 4 above)
Comment (antgonza) [reply qp_target_gene/split_libraries/util.py]: K

File: qp_target_gene/split_libraries/util.py (hunk 2 above)
Comment (antgonza) [reply qp_target_gene/split_libraries/util.py]: K

File: qp_target_gene/split_libraries/util.py (hunk 3 above)
Comment (antgonza) [reply qp_target_gene/split_libraries/util.py]: K

**Review by antgonza — COMMENTED** [review]
Thank you!

File: qp_target_gene/split_libraries/util.py [hunk 5]
```diff
@@ -41,7 +43,39 @@ def get_artifact_information(qclient, artifact_id):
     # Get the artifact metadata
     prep_info = qclient.get('/qiita_db/prep_template/%s/'
                             % artifact_info['prep_information'][0])
-    qiime_map = prep_info['qiime-map']
+
+    df = pd.read_csv(prep_info['prep-file'], sep='\t', dtype='str',
+                     na_values=[], keep_default_na=True)
```
Comment (ElDeveloper) [inline qp_target_gene/split_libraries/util.py:48]:
```suggestion
                     na_values=[], keep_default_na=False)
```

**Review by ElDeveloper — APPROVED** [review]
Looks good, just one change!

**Review by ElDeveloper — APPROVED** [review]

**Comment by ElDeveloper** [comment]
Thanks @antgonza

**Pull request merged**